app/views/hub/organizations/_edit_organization.html.erb

```diff
@@ -1,5 +1,16 @@
 <%= f.cfa_input_field :name, t("general.name") %>
```

Comment:
can we rename this to `_organization_form.html.erb` because it isn't always editing